test/users/engin/compiler/nestedRecordField.future

```diff
@@ -0,0 +1,4 @@
+bug: inner record object as a field
+
+when outer is a record it seems to fail whereas when it is class it seems to be
+working fine
```

Comment:
Okay.  It might be good to call out that this difference in the test code is intentional (especially if you're linking the two tests).  But you don't have to :)